unfurl_message.py

```diff
@@ -194,6 +195,10 @@ def get_merge_requests_info(session, path_info):
         title += f" ({state})"
 
     fields = [{"title": "Assignee", "value": assignee, "short": "true"}]
+    if diffs:
+        fields.append(
+            {"title": "Diffs", "value": diffs, "short": "true"}
```

Comment:
The name 'Diffs' doesn't really explain what this number should show, I expected it would say "+45 -33" or something to count how many differences there are between the two versions.

I'm actually still not sure. Is it the number of commits? If so, could you call them Commits instead?

Reply:
Gitlab's documentation does not mention the full definition of the `changes_count`. And I was sure, that it is a number of total changed lines (additions and deletions).

So the idea was to add the number of the total changes to a preview, but now when I tried it on our merge requests, it looks like `changes_count` is for the number of changed files.

There is a couple of ways fo retrieving actual diffs:
1. Make another request to the [`GET /projects/:id/merge_requests/:merge_request_iid/changes`](https://docs.gitlab.com/ee/api/merge_requests.html#get-single-mr-changes) and parse all `diffs` for the `@@ -11,30 +11,30 @@` values.
2. Get a list of commit references from the call above and make calls for every commit to retrieve changes vie API.

Theoretically, we can implement the first one, but I'm not sure if we want to make twice as many calls only for one new field. 

So I suggest closing this PR because it actually does not implement what I wanted. Or we can rename the field to the `Changes`/`Changed files`/`Files` or anything else. But I'm not sure if the number of the changed files is interesting for someone.